src/renderer/components/layout/LibraryLayout.tsx

```diff
@@ -46,21 +46,22 @@ class LibraryLayout extends React.Component<LibraryLayoutProps, undefined> {
         }
 
         return (
-            <>
+            <div
+                style={ this.props.dialogOpen ? {filter: "blur(1px)"} : {} }
+            >
```

Comment:
if I am not mistaken, this fix is only for the library window, could you please check the reader view as well? See the following screenshot which shows the slide-out right-hand-side menu incorrectly displaying over the modal popup dialog: https://github.com/readium/readium-desktop/issues/359#issuecomment-504683032

Reply:
@Scarsniik said: "Yes, it works with the reader"
https://github.com/readium/readium-desktop/issues/359#issuecomment-507579537

Reply:
Tested by me, works fine in reader view too.